### PR TITLE
Improvement of the zenith interpolation with more control for the user

### PR DESCRIPTION
**Pull request by mdebony**
I've added an option to switch between a linear and logarithmic interpolation for the background model.
I've also added an experimental cleaning procedure to remove bad pixels from the interpolated background model that where the cause of issue when fitting the normalization of the background.
The overall code for interpolation was also a bit reorganized.

### CODE REVIEW

**Review by gabemery — CHANGES_REQUESTED** [review]
Quick review assuming all functionalities are working.

File: acceptance_modelisation/base_acceptance_map_creator.py [hunk 1]
```diff
@@ -753,41 +892,35 @@ def create_acceptance_map_cos_zenith_interpolated(self,
         cos_zenith_model = np.array(cos_zenith_model)
 
         acceptance_map = {}
-        if len(binned_model) <= 1:
+        if len(dict_binned_model) <= 1:
```
[the remark this answers is not in the document]
Comment (mdebony) [reply acceptance_modelisation/base_acceptance_map_creator.py:895]: True, this is not needed anymore with the refactoring.

File: acceptance_modelisation/base_acceptance_map_creator.py [hunk 2]
```diff
@@ -20,7 +20,8 @@
 
 from .bkg_collection import BackgroundCollectionZenith
 from .exception import BackgroundModelFormatException
-from .toolbox import compute_rotation_speed_fov, get_unique_wobble_pointings, get_time_mini_irf, generate_irf_from_mini_irf
+from .toolbox import compute_rotation_speed_fov, get_unique_wobble_pointings, get_time_mini_irf, \
```
[the remark this answers is not in the document]
Comment (mdebony) [reply acceptance_modelisation/base_acceptance_map_creator.py]: Done

**Review by gabemery — APPROVED** [review]
Approved

**Pull request merged**